### DISCUSSION TITLE
Welcome to Cardinal Discussions!

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TuonyBT** [comment]
My name is Tony. I am a music lover with an itch to make my own but little talent or application. However, I do like to play with stuff. I have played guitars on and off for 35 years, but spent more time taking things apart than playing actual music. I also like programming and otherwise messing with computers, love open-source and have used any number of GNU-Linux flavours on my personal machines for ages.

I am looking to modular synthesis as a means to bring all these things together, and a learning path that can be both fun and challenging. I wanted to start with ARP2600 and quickly found that I could 'build' one with Cardinal. So here I am.

I hope it will be helpful to others if I share some of my experiences. In due course I hope to become an experienced user and perhaps at some point make a contribution to the project.

**Comment by TuonyBT** [comment]
Initial experience using AV Linux, Ardour as a DAW and Carla as a plugin host. Managed to copy Cardinal into my VST3 and LV2 folders, and both Ardour and Cardinal have picked them up. I have not yet managed to get Ardour's internal keyboard, or the JACK keyboard, to trigger anything but everything seems to work fine with Carla. I suspect my problem is with Ardour/JACK, not with Cardinal.

I have built a couple of simple patches, following along with Youtubers Venus Theory and Monodux. Thanks to the latter I have a platform on which to follow ARP2600 lessons. My plan now is to work through Samuel Ecoff's 'Fundamentals of Music Technology', building up my ARP2600 patch as I need new modules for each exercise. I will do my best to share any observations that I think may help with the development of Cardinal.
